### DISCUSSION TITLE
latest version

### DISCUSSION THREAD

**Discussion by VirulentSlayer** [aclap-dev/video-downloadhelper#1064] [Help and Q&A] [answered]
### Your question

There appears to be a new problem with your latest build of the extension for Firefox, I believe i'm on the same revision as last week but now it is failing to download any videos, even the same videos as ones which works just days ago without a problem. 

I couldn't find a link or download for beta as mentioned in previous threads, but I've managed to downgrade to a v.8.x and this has reenabled functionality.

For those who are asking for a refund, might I suggest that you hold out for a new fix or downgrade to v8 to continue getting the most of your purchase.

 (Video DownloadHelper 8.2.2.8 (stable) is working for me as we speak.) 
 a link for Firefox users. https://addons.mozilla.org/en-US/firefox/addon/video-downloadhelper/versions/
 
 Thanks

### Addon & CoApp version and details

⊖
Video DownloadHelper 8.2.2.8 (stable)
Video DownloadHelper
Version 8.2.2.8 (stable)
Browser locale: en-GB
Production build
Built on 2024-03-18
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla
⊖
Platform
Platform Win x86-64
Browser Mozilla Firefox 128.0.2
⊖
Companion App installed
Found companion app: VdhCoApp 2.0.19
Companion app binary: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

⊖
License verified

**Comment by GrampaWildWilly** [comment]
The beta is in the Wiki.  There's a link to the Wiki on every page of this forum.  Click that link, then look for the reference to beta versions.  Click that.  You'll find the beta.

**Comment by VirulentSlayer** [comment]
Thank you.

**Reply by GrampaWildWilly** [reply]
You're quite welcome, but you should hold your thanks until you actually get the beta installed & running.  Do let us know if it solved your problem, which by the way you've been rather vague about.  If your problem persists, do post at least one if not a few URLs of web pages where there are problematic videos.  That's so others can try the same thing & we can all compare results.  Also, click the `Copy information to clipboard` button in the VDH settings & post whatever that gives you.  There might be useful bits of information buried in there.

**Reply by VirulentSlayer** [reply, marked as answer]
I have found the problem. It was my own fault, no such directory kind of problem. I feel very silly now. I have diagnosed this using an older version which gave an log upon failure, Version 8.2.0.23 I believe. 

This log is not in the later versions and could help some users like it has myself. 

Thanks again!

**Reply by GrampaWildWilly** [reply]
Ah yes.  Error reporting.  I have complained about that recently.  The only error we get now goes along the lines of, "We've had an error.  Please click this button to report it."  And then the report goes somewhere mysterious & some time later, the solution might find its way into a new release.  I say that's not good enough.  Whatever is getting reported should be visible to us.  As your case proves so eloquently, sometimes we might be able to look at the problem report & figure out what's wrong without reporting it to the mystery destination.  Oh well.  Maybe some day we'll get this improved.

Meanwhile, did you install the beta & get it working?  It's always best to be on the latest versions because you won't get any support on antique versions.  If you hit a problem on an antique version, you know the answer here is going to be that you should get on the current version & then we can talk.

**Reply by VirulentSlayer** [reply]
I did install the beta successfully and inevitably encountered the same problem due to settings I had previously configured, that is when i reverted to an older version and discovered that it produced a log while I could further examine. 

After realising it was not a problem with the extension itself but a problem with my configuration I made adjustments and I have been using the latest current version since. 

I am happy to confirm there are no problems.

Thank you again for your time and responses.
